test/test_prototype_transforms.py

```diff
@@ -437,7 +437,7 @@ def test__get_params(self, fill, side_range, mocker):
         image = mocker.MagicMock(spec=features.Image)
         h, w = image.spatial_size = (24, 32)
 
-        params = transform._get_params(image)
+        params = transform._get_params([image])
```

Comment:
All changes in this file stem from the fact that `_get_params` previously handled pytree objects, but now only handles flattened ones.